### DISCUSSION TITLE
Use time log for report builder

### DISCUSSION THREAD

**Discussion by danbroun** [shridhar-tl/jira-assistant#247] [Q&A]
Is it possible to use time log for in the report builder? The time log I can currently export does not contain all the fields which are of interest for me.

**Comment by shridhar-tl** [comment]
Aah, I somehow missed this conversation. Yes it is possible to pull any information from Jira using Report Builder, however, you need to write your own logic to render it as you need. However what is that field your are expecting which is not available in current worklog report?

**Comment by templarvii** [comment]
We have the same requirement.

Our Jira tickets have a custom field "Job Number" that cannot be added by the "Worklog Report". So, we have tried to create a custom report in "Report Builder" but we have not found a way to include the "Work log".

Do we need to use "Flattern Dataset"?
Is there a documentation how to use the "Flattern Dataset"?

**Comment by shridhar-tl** [comment]
@danbroun  @templarvii - Apologies for delayed response. Now, starting from JA v2.49, you can add any custom field to worklog report and hope that resolves your issue. Please have a look at #294 for more details.

Closing this discussion hoping your issue is resolved as part of this implementation. Please feel free to reopen it if you have any other queries.
